src/openeo_grass_gis_driver/processes_process_id.py

```diff
@@ -15,10 +15,10 @@
 __email__ = "soerengebbert@googlemail.com"
 
 
-class ProcessesProcessId(ResourceBase):
+class ProcessesProcessId(Resource):
```

Comment:
This will make the endpoint open to everyone. Why not leave the `ResourceBase` overwrite with authentication?

Reply:
I wanted to look it up in the documentation - it seems as this is not part of it anymore? I would like to keep it, at least until the /processes endpoint returns the full description of each process.
Ok I wasn't sure if we agreed on the advertising already. Then we leave/make it open and I resolve this thread.